WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2016,7 +2016,7 @@ static GSourceFuncs _handlerIntervention =
 
             _adminLock.Unlock();
         }
-        void OnLoadFailed()
+        void OnLoadFailed(const string _URL)
```

Comment:
This could be very confusing, i mean shadowing the class property with an argument. Lets make this explicit and make a change in notification call bellow.  i.e. name the parameter as 'URL' and pass it to the notification handler below.